src/entrypoint.sh

```diff
@@ -9,7 +9,11 @@ invoke () {
     then
         /usr/local/bin/invoke $@
     else
-        /usr/local/bin/invoke $@ >> /usr/src/{{project_name}}/invoke.log 2>&1
+        if [ $IS_CELERY = 'true'] || [ $IS_CELERY = 'True' ]
```

Comment:
What's the purpose of this new block? 
Each container already has its own local invoke log file `/usr/src/{{project_name}}/invoke.log`, why are you differentiating the name in `/usr/src/geonode_proj/invoke.log` for GeoNode and `/usr/src/{{project_name}}/invoke.log` for Celery?

Reply:
I deemed them as separate log files and checking whether the log output comes from a celery container or not distunguished by `IS_CELERY` env variable.

Reply:
these logs are saved locally inside the container (`/usr/src/{{project_name}}/invoke.log`), so we already know which container they refer to. These are distinct from the actual logs, that are sent to `/var/log`.